$MirOS: ports/devel/cvs/patches/patch-src_status_c,v 1.1 2010/09/15 20:57:03 tg Exp $
--- src/status.c.orig	Thu Sep 22 18:38:06 2005
+++ src/status.c	Sat Sep 18 23:37:04 2010
@@ -15,6 +15,8 @@
 
 #include "cvs.h"
 
+__RCSID("$MirOS: ports/devel/cvs/patches/patch-src_status_c,v 1.1 2010/09/15 20:57:03 tg Exp $");
+
 static Dtype status_dirproc (void *callerdat, const char *dir,
                              const char *repos, const char *update_dir,
                              List *entries);
@@ -213,8 +215,11 @@ status_fileproc (void *callerdat, struct
 	cvs_output ("   Working revision:\t", 0);
 	cvs_output (vers->vn_user, 0);
 
-	/* Only add the UTC timezone if there is a time to use. */
-	if (!server_active && strlen (vers->ts_rcs) > 0)
+	/* Only add the UTC timezone if there is a time to use.
+	 * ts_rcs sometimes contains only "=" character so we check len > 1 */
+	if (!server_active && strlen (vers->ts_rcs) > 1 &&
+	    /* prevent an ugly error message */
+	    strcmp(vers->ts_rcs, "Result of merge") != 0)
 	{
 	    /* Convert from the asctime() format to ISO 8601 */
 	    char *buf;
